ui/src/app/completed/controllers/TableCtrl.js

```diff
@@ -6,20 +6,71 @@
 
   function TableCtrl($scope, ToastService, TableService, UtilityService) {
     $scope.downloads = [];
-
+    var fileTypeArray = ['_blank', '_page', 'aac', 'ai', 'aiff', 'avi', 'bmp',
```

Comment:
Shall we move this to a separate file and import to here? Then someone else can reuse that in somewhere else.

Reply:
sure, good idea

Reply:
should we include that file in global scope or use angular scope. I think keeping Js built-in variable would more better as it's static

Reply:
How does this one look?


![image](https://user-images.githubusercontent.com/25587962/40937269-afb13d54-685b-11e8-8eb2-2fa1797c23fa.png)

Reply:
Done 👍